lib/lunchbot/application.ex

```diff
@@ -7,7 +7,7 @@ defmodule Lunchbot.Application do
 
   @impl true
   def start(_type, _args) do
-    unless Mix.env() == :prod do
+    unless Application.get_env(:lunchbot, :env) == :prod do
       Dotenv.load()
       Mix.Task.run("loadconfig")
     end
```

Comment:
I was assuming the dotenv was only being used in dev environments (local dev and test), so I was going to keep that in place for now.

Reply:
You could still have an env check in `config/runtime.exs`

```ex
# In `config/runtime.exs`
unless config_env() == :prod do
  Dotenv.load()
  Mix.Task.run("loadconfig")
end
```

I see the readme for dotenv recommends placing them in Application.start but it also says it's not compatible with environments without Mix. Runtime config seems to me like it would be a better place to put it. And it's relatively new, so I wouldn't be surprised if the dotenv readme was just out of date. Actually, I just looked it up and there is an issue in the github repo along these lines: https://github.com/avdi/dotenv_elixir/issues/38

Reply:
> I assume you're using elixir releases since mix isn't available

Haven't used it before but I think running `mix release` in a dev environment creates a standalone app for production that might package mix?

Reply:
It creates a standalone compiled app that does _not_ include `Mix` (or any other `:dev` dependencies).